README.md

```diff
@@ -155,17 +156,56 @@ The default is 70 characters for each of these:
 }
 ```
 
+### new-line-at-eof
+
+`new-line-at-eof` can also be configured to enforcing or disallowing new lines at EOF.
+- To enforce new lines at EOF:
+```
+{
+  "new-line-at-eof": ["on", "yes"]
+}
+```
+- To disallow new lines at EOF:
+```
+{
+  "new-line-at-eof": ["on", "no"]
+}
+```
+
+### no-dupe-scenario-names
 
-### no-restricted-tags
```

Comment:
You deleted the `no-restricted-tags` rule (probably a merge conflict) documentation. Do you mind re-adding it?